test/conftest.py

```diff
@@ -57,9 +57,9 @@ def maker(content, **kwargs):
         from test.utils import *
         from hypothesis import settings
         raw_schema = {schema}
-        schema = schemathesis.from_dict(raw_schema)
+        schema = schemathesis.from_dict(raw_schema, method={method}, endpoint={endpoint})
         """.format(
-                schema=schema
+                schema=schema, method=repr(method), endpoint=repr(endpoint)
```

Comment:
Why is `repr` needed?

Reply:
To have `method="GET"` instead of `method=GET` after format.

Reply:
And `method=None` instead of `method="None"` for hardcoded quotes.

Reply:
I'd suggest using double quotes in the template instead

Reply:
Update: in this case `None` will become `"None"`